seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-mysql-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/cdc/mysql/MysqlCDCIT.java

```diff
@@ -360,7 +360,8 @@ public void testMultiTableWithRestore(TestContainer container)
 
         Pattern jobIdPattern =
                 Pattern.compile(
-                        ".*Init JobMaster for Job SeaTunnel_Job \\(([0-9]*)\\).*", Pattern.DOTALL);
+                        ".*Init JobMaster for Job mysqlcdc_to_mysql_with_multi_table_mode_one_table.conf \\(([0-9]*)\\).*",
+                        Pattern.DOTALL);
```

Comment:
**Issue**: The job name in the regex pattern has been hardcoded to a specific test configuration file name. This could lead to maintenance issues if the configuration file names are changed or new configurations are added. <br> **Fix**: Use a more generic pattern or derive the pattern dynamically from the configuration file name to avoid hardcoding. <br> **Code Suggestion**: 
 ```
 - ".*Init JobMaster for Job mysqlcdc_to_mysql_with_multi_table_mode_one_table.conf \\\\(([0-9]*)\\\\).*",
 + ".*Init JobMaster for Job " + getConfigFileName() + " \\\\(([0-9]*)\\\\).*"
 ```